src/skimage/registration/_affine.py

```
@@ -0,0 +1,1180 @@
+                    options=options,
+                )
+
+                # If cost less than minimum cost, update
+                if result.fun < fvalmin:
+                    fvalmin, x = result.fun, result.x
+
+                # print(f"{trial}:ok  -> {result.fun} {result.fun <= fvalmin}")
+                # stop after a few successful run
+                if trial == self.min_trial - 1 and fvalmin != np.inf:
+                    break
+            except Exception as _:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)